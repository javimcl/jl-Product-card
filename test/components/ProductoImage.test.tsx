import React from 'react';
import renderer from 'react-test-renderer';
import ProductCard, { ProductoImage } from '../../src/components'
import { product2 } from '../data/products';


describe('Funcionamiento de <ProductImage/>', () => { 
    test('debe de funcionar correctamente', () => { 

        const wrapper = renderer.create(
            <ProductoImage img='http:dddd.png'/>
          )

          expect(wrapper.toJSON()).toMatchSnapshot();

     })

     test('debe  de mostar el componente con la imagen del producto', () => { 
        const wrapper = renderer.create(
            <ProductCard product={product2}>
              {
                () => (
                  <ProductoImage />
                )
      
              }
      
            </ProductCard>
          )
          expect(wrapper.toJSON()).toMatchSnapshot();
      })
 })