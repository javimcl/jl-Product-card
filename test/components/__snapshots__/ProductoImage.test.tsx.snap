// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Funcionamiento de <ProductImage/> debe  de mostar el componente con la imagen del producto 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product "
    className="productImg undefined"
    src="./coffee-mug2.png"
  />
</div>
`;

exports[`Funcionamiento de <ProductImage/> debe de funcionar correctamente 1`] = `
<img
  alt="Product "
  className="productImg undefined"
  src="http:dddd.png"
/>
`;
